results/cmd/api/main.go

```diff
@@ -57,7 +59,24 @@ type server struct {
 	db *sql.DB
 }
 
-func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.TaskRun, error) {
-	// TODO: implement this.
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTaskRun not implemented")
+// Receive TaskRun from Watcher and save it to local Sqlite Server
+func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.CreateTaskRunResponse, error) {
+	database := s.db
+	statement, _ := database.Prepare("CREATE TABLE IF NOT EXISTS taskrun (logid INTEGER PRIMARY KEY, taskrunlog BLOB)")
+	statement.Exec()
+	statement, _ = database.Prepare("INSERT INTO taskrun (taskrunlog) VALUES (?)")
```

Comment:
For this and other statements, make sure we check and return the error.

Reply:
Sure